esp-hal/src/i2c.rs

```diff
@@ -2098,8 +2019,40 @@ where
     I: Iterator<Item = &'a COMD>,
 {
     let cmd = cmd_iterator.next().ok_or(Error::CommandNrExceeded)?;
-    let cmd_reg: CommandReg = command.into();
-    cmd.write(|w| unsafe { w.bits(cmd_reg.bits()) });
+    unsafe {
+        match command {
+            Command::Start => {
+                cmd.write(|w| w.opcode().bits(OPCODE_RSTART as u8));
```

Comment:
When https://github.com/esp-rs/esp-pacs/pull/261 is merged it can be just
```rust
                cmd.write(|w| w.opcode().rstart());
```
or
```rust
use pac::i2c0::comd::Opcode;

                cmd.write(|w| w.opcode().variant(Opcode::Rstart));
```

Reply:
Oh, right, I made a "variant" out of it, totally forgot about it!

Good catch, will be updated, thanks!